src/saleor_app/app.py

```diff
@@ -53,6 +63,20 @@ def include_webhook_router(self):
                 401: {"description": "Incorrect signature"},
                 404: {"description": "Incorrect saleor event"},
             },
+            dependencies=[
```

Comment:
Those are the required dependencies (which are not needed withing the handlers).